src/cycax/cycad/engines/shared_openscad.py

```diff
@@ -0,0 +1,37 @@
+# Functions shared between the OpenSCAD based AssemblyEngine and PartEngine
```

Comment:
This is not for this PR. Once the OpenSCAD Assembly has moved here. We should move shared code into this file. I expect that it will hold all the string processing related to OpenSCAD, The SCAD code creation all in one place.